flutter_nsd.iml

```diff
@@ -23,7 +23,6 @@
       <excludeFolder url="file://$MODULE_DIR$/example/ios/.symlinks/plugins/flutter_nsd/build" />
     </content>
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Dart Packages" level="project" />
```

Comment:
Not sure why this was removed

Reply:
Not intentionally. this is an ide config file.